src/components/NoWallet/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import ButtonCTA from 'components/ButtonCTA'
+
+interface NoWalletProps {
+  handleClick?(): void
+  hide?: boolean
+}
+
+export const NoWallet: React.SFC<NoWalletProps> = ({ 
+  handleClick,
+  hide = false, 
+}): JSX.Element => (
```

Comment:
`JSX.Element` is unnecessary as `React.SFC` provides everything we need, including return values

Reply:
cool thanks